src/middleware.js

```diff
@@ -4,7 +4,7 @@ import http from './http'
 import { lpApiRequest, lpApiSuccess, lpApiFailure } from './actions'
 
 const DEFAULT_CONFIG_OPTIONS = {
-  onUnauthorized: undefined,
+  onUnauthorized: undefined
```

Comment:
we typically have trailing commas, David is a big fan, either way, no need to remove them.